scrapers/wearehairy.yml

```
@@ -0,0 +1,25 @@
+name: wearehairy
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - wearehairy.com
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //h1[@itemprop="name"]//h1
+      Date:
+        selector: //span[@class="added"]//@datetime
+        postProcess:
+          - parseDate: 2006 Jan 02
```

You forgot to edit the parseDate
Its the same format that we use so we could skip the postProcess altogether
Just to  be safe though
```suggestion
          - parseDate: 2006-01-02
```